css/style.css

```
@@ -0,0 +1,150 @@
+/*=============================================================================
+ベース
+==============================================================================*/
+*, *::before, *::after {
+  box-sizing:inherit;
+}
+
+html {
+  box-sizing:border-box;
+}
+
+body {
+  color:#333;
+  font-family:'Roboto','sans-serif';
+  line-height:1.8;
+  font-size:14px;
+}
+
+img {
+  width:100%;
+  height:auto;
+  vertical-align:bottom;
+}
+
+a {
+  text-decoration:none;
+  color:inherit;
+}
+
+a:hover {
+  opacity:0.7;
+}
+/*=============================================================================
+グローバル
+==============================================================================*/
+.btn {
+  display:flex;
+  justify-content:center;
+  align-items:center;
+  background-color:#ffa800;
+  color:#fff;
+  border-radius:9999px;
+  border-style:none;
+  cursor:pointer;
+  font-weight:bold;
+}
+
+.pcNone {
+  display:none;
+}
+
+/*============================================================================*/
+header {
+  background-color:#fff;
+  position:sticky;
+  top:0;
+  box-shadow:rgba(0, 0, 0, 0.3) 1px 1px 3px;
+}
+
+header .inner {
+  max-width:980px;
+  width:96%;
+  height:60px;
+  display:flex;
+  justify-content:space-between;
+  align-items:center;
+  margin:auto;
+}
+
+header .logo {
+  width:100px;
+  height:20px;
+}
+
+header .logo img {
+  vertical-align:top;
```

申し訳ないです。指摘指示誤りです。
```suggestion
  display:flex;
```
で調整できます。

---

> 申し訳ないです。指摘指示誤りです。
> 
> で調整できます。

修正
f85ae890ec22b107ccd050ce77d830e630c27b35